src/app/app.component.spec.ts

```diff
@@ -14,6 +14,7 @@ describe('AppComponent', () => {
   const MockUserService = {
     getUserName$: jest.fn().mockReturnValue(new Observable()),
   };
+  (window as any).gtag = () => {};
```

Comment:
Can we suppress this error?

Reply:
showing a lint error `Unexpected any. Specify a different type`

Reply:
resolved, thanks